### DISCUSSION TITLE
Marketing for Krux

### DISCUSSION THREAD

**Discussion by kkdao** [selfcustody/krux#82] [Ideas]
It's great to see this project coming along slowly but surely from a code and docs perspective. WRT docs, I think these are nice and clean pages for anyone curious about the project and what's involved: https://jreesun.github.io/krux/ 👏 

As for your community channel I think you should try Telegram, let me explain. While I understand Matrix is better for privacy, it's just not used by many people. Joinmarket tried to stay on IRC, but now their community has created two Telegram chats that are getting more and more active, and even Waxwing is in both chats now. Yes Telegram has it's downsides with bots/scammers, but Bitcoiners should always be weary of this. A good bot, a couple mods (when it gets bigger), and an informative pinned message for newcomers to read when they join could easily keep the chat in check.

So since you're an engineer and not a marketer @jreesun I propose you create a group chat with you as the owner and you can choose a mod to help you manage it. OR you can send curious users to the DIYbitcoin Telegram group I own [chat](https://t.me/diybitcoin) where some of us have already been chatting/sharing info about the project in there. 

I'm not a marketer either by the way, I just like the project and want to help how I can.

**Comment by ghost** [comment]
> It's great to see this project coming along slowly but surely from a code and docs perspective. WRT docs, I think these are nice and clean pages for anyone curious about the project and what's involved: https://jreesun.github.io/krux/ 👏

Thank you! The eventual goal is to give people a choice between building the entire thing themselves or using an official signed release, so it should hopefully become less intimidating to run over time.

> OR you can send curious users to the DIYbitcoin Telegram group I own [chat](https://t.me/diybitcoin) where some of us have already been chatting/sharing info about the project in there.

I think I will take you up on that offer. I don't want to create a Telegram group myself, but I'm happy to direct people to yours or even a community-created Krux group if it happens. For now, I'll update the docs to point people to the DIYbitcoin group.

> I'm not a marketer either by the way, I just like the project and want to help how I can.

That's awesome to hear, and thank you 😄

**Comment by kkdao** [comment]
That's a great goal, I'm sure many new DIYers would appreciate that, as well as some steps on how to verify the official signed release 👍 

Sounds good! We can start with the DIYbitcoin chat, and I was thinking a community Krux group could be created as soon as it starts becoming too busy in that chat. Or I could start a separate one now, up to you.

**Reply by ghost** [reply]
I'm happy to stick with the DIYBitcoin chat for now. If it does get noisy, you have my blessing to make a Krux-specific one 👍

**Reply by ghost** [reply]
@odudex @kkdao 
Would either of you be open to creating and running a community Krux telegram channel?

**Comment by odudex** [comment]
I've created a group for Brazilians: https://t.me/KruxBR since some of users don't speak English. It's just a few people.
I don't know any Telegram user accounts of Krux users outside Brazil to start another group, if you tell me some I could start a EN group too.

**Reply by kkdao** [reply]
You can add my Telegram account (@fullnodecustoms) and I can help out where I can. Obviously you are much more involved in the project than I have been. So any questions from new users you will mostly have to handle yourself, but the way I can help is I can put together a FAQ list as new questions get asked and then eventually create a doc. That way you would ideally have to answer less and less questions over time as we can point them towards that.

**Reply by odudex** [reply]
Right, group created: https://t.me/SC_Krux
New Kruxers are popping up. We sure will need more documentation and FAQ.

When you accept the invitation I'll put you as admin too
